src/app/core/locale/server-locale.service.ts

```diff
@@ -53,7 +53,7 @@ export class ServerLocaleService extends LocaleService {
    *
    * @returns {Observable<string[]>}
    */
-  getLanguageCodeList(): Observable<string[]> {
+  getLanguageCodeList(ignoreEPersonSettings = false): Observable<string[]> {
```

Comment:
@AbhinavS96 : It appears this new setting isn't being used in this backport.  Shouldn't this be used below on line 66 just like in #4805?